senders/beanstlakd_sender.go

```diff
@@ -37,8 +37,23 @@ func (c *BeanstalkdSender) SendEvent(e models.IncomingEvent, d models.Destinatio
     return err
   }
 
-  pri, delay, ttr := uint32(1), time.Duration(0), time.Duration(0)
-  id, err := conn.Put(jobBody, pri, delay, ttr)
+  var pri uint32 = 0
+  var delay time.Duration
+  var ttr time.Duration
```

Comment:
let's give some default values to pri, delay and ttr

Reply:
resolved at https://github.com/shoplineapp/captin/pull/44/commits/0925b5053cebf99c87ba583b64d3221b8c2b4118